fontforge/splinestroke.c

```diff
@@ -919,19 +1029,38 @@ static void SSAppendArc(SplineSet *cur, bigreal major, bigreal minor,
 
     _CalcNibOffset(nc, n, ut_fm, false, &no_fm, -1);
     _CalcNibOffset(nc, n, ut_to, false, &no_to, -1);
+    if ( limit ) {
+	if (    !bk && no_fm.nci[false] == no_to.nci[false]
```

Comment:
That's a 2 element array that corresponds to an "is_ccw" condition, where I'm picking one value arbitrarily because I know the two values won't be different in this case. I'm using "false" because of that pattern but I'm happy to just use 0 instead.